pallets/pallet-bonded-coins/README.md

```diff
@@ -1,4 +1,137 @@
-# pallet bonded coins
+# Pallet Bonded Coins
 
-Create currencies which can be minted by locking up tokens of an existing currency as collateral.
-The exchange rate of collateral to tokens minted is determined by a bonding curve linking total issuance of the new currency to mint price.
+The Pallet Bonded Coins module allows for the creation of new currencies that can be minted by locking up tokens of an existing currency as collateral. The exchange rate of collateral to tokens minted is determined by a bonding curve that links the total issuance of the new currency to its mint or burn price.
+
+## Overview
+
+This pallet provides functionality to:
+- Create new currency pools with specified bonding curves and collateral types.
+- Mint new tokens by locking up collateral.
+- Burn tokens to release collateral.
+- Manage the lifecycle of currency pools, including refunding and destroying pools.
+
+### Rounding 
+
+Rounding issues are a problem and cannot be completely avoided due to the nature of limited resources on a computer, resulting in a lack of representation for irrational numbers. This pallet cannot guarantee mathematically exact calculations. However, it can guarantee the reproducibility of the same result based on the usage of [fixed-point][fixed-point] numbers. 
+
+
+## Key Concepts
+
+### Bonding Curve
+A bonding curve is a mathematical curve that defines the relationship between the supply of a token and its price. In this pallet, the bonding curve determines the cost of minting or burning tokens based on the current supply. The current system implements the LMSR, square root, and polynomial bonding curves.
+
+- LMSR (Logarithmic Market Scoring Rule): A market-making algorithm that adjusts prices based on the logarithm of the token supply.
```

Comment:
Link to this here?

Reply:
It should be added to the first time the acronym shows up. So if this is the first time, then it should be here instead.

Reply:
Yeah, it's here the second time. I added a link on the first occurrence: https://github.com/KILTprotocol/kilt-node/pull/807/files#diff-b5e6eaa977a447df87f2e825f0da7d5aee88374aea0d0e87878523a14422a70bR26